src/header/context.rs

```diff
@@ -12,14 +10,14 @@ pub enum ContextLabel {
     When,
 }
 
-impl From<ContextLabel> for &'static str {
-    fn from(label: ContextLabel) -> Self {
-        match label {
-            ContextLabel::Describe => "Describe",
-            ContextLabel::Context => "Context",
-            ContextLabel::Specify => "Specify",
-            ContextLabel::Given => "Given",
-            ContextLabel::When => "When",
+impl fmt::Display for ContextLabel {
```

Comment:
This is not blocking, just a thought about design.

When I do it, I like to:
1) keep the `From<*Label> for &str`
2) unit test the From<Label> to &str
3) Display is a simple call to `write!(f, "{}", str::from(label))` as IO is nasty for testing (no real problem with this particular case. Just a general remark)

Again, not bloacking.

Reply:
Yeah. I couldn't think of a single use case where one might want to access the label as string representation, other than for the internal use for logging, which already makes use of `Display`/`Debug`. Also with the method being dead-simple other than pure test-coverage for the sake of test coverage I couldn't find a good reason to make things more complicated than necessary.

Reply:
good for me ! 👍